app/styles/front.scss

```diff
@@ -688,21 +690,51 @@ $autre-lighter: #cce4e7;
   cursor: help;
 }
 
-[data-toggle="buttons"] {
-  display: flex;
-  flex-direction: row;
+legend {
+  border-bottom: 0px;
+}
 
-  button, label {
-    flex: 1;
-  }
+.radioButton {
+  @extend .btn-default;
+}
 
-  .active .help-block, .active .help-col {
-    color: #D8D8D8;
-  }
+.invisibleInput {
```

Comment:
Darn, `invisible` is a built-in bootstrap class with `visibility: hidden`. This breaks the accessibility mechanism (the form fields are ignored by the screen reader).